web/src/components/AssertionForm/Fields/referencePicker.tsx

```diff
@@ -0,0 +1,8 @@
+import AttributesTags from '../../../constants/AttributesTags.json';
+import {OtelReference} from '../hooks/useGetOTELSemanticConvertionAttributesInfo';
+
+const attributesTags: Record<string, {tags: string[]; description: string}> = AttributesTags;
+
+export function referencePicker(reference: OtelReference, key: string) {
```

Comment:
I think we can move this to the `SpanAttribute` service isn't it?

Reply:
ok